### ISSUE TITLE
Blur datepicker after date selection

### ISSUE THREAD

**Issue by al3x-huang** [Hacker0x01/react-datepicker#771]
Is there any possible way to force datepicker input to blur (lose focus) after a date has been selected? It is annoying because once the datepicker has been clicked it keeps focus even though it is closed so when you switch windows or tabs then return the datepicker will open by itself.

**Comment by alex-shamshurin** [comment]
You can move focus to another input field or whatever  in `onChange` callback.

**Comment by nbkhope** [comment]
@alex-shamshurin can you elaborate?

**Comment by kunalshewale** [comment]
@al3x-huang : Did you got solution?

**Comment by adam-horvath** [comment]
For me, a zero-time setTimeout was the trick:
`setTimeout(() => document.activeElement.blur());`

**Comment by pratibha-negi-asx** [comment]
I've tried using: document.activeElement.blur() and document.body.focus() in onChange event handler of DatePicker, but still the focus remains in DatePicker. I'm using Custom Date Picker here.
Snippet of my code:
<CustomDatePicker  onChange={this.handleStartDateChange} />

handleStartDateChange = (newValue) => {
    console.log("document ", document.activeElement)
    document.activeElement.blur()
    //document.body.focus();
    console.log("document ", document.activeElement)
}

One interesting thing is on consoling log of document.activeElement. After blur() is being called the activeElement is logged as body, but the focus remains on DatePicker.
The issue surfaced when I've upgraded react-datepicker library to version: "react-datepicker": "^6.6.0"
It was working fine in version 2.9.6. By fine I mean that the DatePicker was losing focus when the date is selected.

Can somebody help me here.
